src/main/java/com/segment/analytics/android/integrations/adjust/AdjustIntegration.java

```diff
@@ -92,15 +92,15 @@
     super.identify(identify);
 
     String userId = identify.userId();
-    if (userId != null) {
+    if (!isNullOrEmpty(userId)) {
```

Comment:
This is a good example of why pulling the code into a helper function is a good idea. Instead of keeping it up to date in multiple places, you'll only have to make changes in one place.